addons/block_code/ui/blocks/utilities/background/background.gd

```diff
@@ -1,20 +1,22 @@
 @tool
 extends Control
 
-const KNOB_X = 10.0
-const KNOB_W = 20.0
-const KNOB_H = 5.0
-const KNOB_Z = 5.0
+const Constants = preload("res://addons/block_code/ui/constants.gd")
 
 @export var color: Color:
 	set = _set_color
 
+@export var outline_color: Color:
+	set = _set_outline_color
+
```

Comment:
MS Arcade has them separated and I'm planning to borrow them :)

Reply:
Oh ok, sounds good!

Reply:
I just looked at the HTML, for one they are using Blockly to render the blocks. And there is no pre-set amount that the borders are darkened, but it is in the range of 14-16% and sometimes the saturation changes a little bit. I don't think it would be too hard to implement automatic darkening/hue shifting and that way we wouldn't need to specify a another new color each time we add a category. Up to you though.